lib/game.js

```diff
@@ -0,0 +1,40 @@
+class Game {
+  constructor () {
+    this.frames = []
+  }
+
+  addRolls = (roll1, roll2, bonusRoll = 0) => {
+    this.maxTen(roll1, roll2)
+    if (this.isLastFrame() && this.isSpare(roll1, roll2) || this.isLastFrame() && this.isStrike(roll1, 0)) {
+      this.frames.push([roll1, roll2, bonusRoll])
```

Comment:
I think this is the bit saying to add a bonus roll if it's the last frame and a spare/strike is bowled? Not fully sure however - it seems like this method is more adding the rolls than this